src/daemons/MetaDaemon.cpp

```diff
@@ -100,14 +100,16 @@ int main(int argc, char *argv[]) {
     // The meta server has only one space, one part.
     partMan->addPart(0, 0, std::move(peersRet.value()));
 
-    // folly IOThreadPoolExecutor
+    // notice: ioThreadPool and acceptThreadPool will stopped when NebulaStore free raftservice
     auto ioPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_num_io_threads);
+    auto acceptThreadPool = std::make_shared<folly::IOThreadPoolExecutor>(1);
```

Comment:
good point, I think about this problem, but I get the result is there is no need configure, 1 thread is too much, because accept thread pool just accept connection from client which is our process like graphd or meta, and they a long connection, so the performance is not high, what do you think about?